cristin-biobank/src/main/java/no/unit/nva/biobank/model/cristin/CristinBiobank.java

```diff
@@ -0,0 +1,199 @@
+package no.unit.nva.biobank.model.cristin;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.databind.PropertyNamingStrategies;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import no.unit.nva.biobank.model.nva.Biobank;
+import no.unit.nva.commons.json.JsonSerializable;
+import no.unit.nva.cristin.model.CristinExternalSource;
+import no.unit.nva.cristin.model.CristinOrganization;
+import no.unit.nva.cristin.projects.model.cristin.CristinApproval;
+import no.unit.nva.cristin.projects.model.cristin.CristinDateInfo;
+import no.unit.nva.cristin.projects.model.cristin.CristinPerson;
+import nva.commons.core.JacocoGenerated;
+
+import java.beans.ConstructorProperties;
+import java.time.Instant;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.Set;
+
+@SuppressWarnings({"PMD.ExcessiveParameterList","PMD.TooManyFields"})
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonNaming(PropertyNamingStrategies.SnakeCaseStrategy.class)
+public class CristinBiobank implements JsonSerializable {
+    // https://api.cristin-test.uio.no/v2/doc/index.html#GETbiobanks
+
+    private final String cristinBiobankId;
+    private final String type;
+    private final Map<String, String> name;
+    private final String mainLanguage;
+    private final Instant startDate;
+    private final Instant storeUntilDate;
+    private final String status;
+    private final CristinDateInfo created;
+    private final CristinDateInfo lastModified;
+    private final CristinOrganization coordinatingInstitution;
+    private final Map<String, String> statusName;
+    private final Map<String, String> typeName;
+    private final CristinPerson coordinator;
+    private final Set<CristinExternalSource> externalSources;
+    private final List<CristinApproval> approvals;
+    private final List<CristinBiobankMaterial> biobankMaterials;
+    private final String biobankId;
+    private final CristinAssociatedProject associatedProject;
+
+
+    @ConstructorProperties({"cristinBiobankId", "type", "name", "mainLanguage", "startDate", "storeUntilDate", "status",
```

Comment:
Same here about snake case / camel case . Do you have an integration karate test?

Reply:
yes 
cristin-biobank/src/test/resources/cristinBiobank1.json